Clarify host OS support (#90901)
Azure/azure-content: articles/virtual-desktop/remote-app-streaming/licensing.md

  - Paid in advance through a subscription
  - Same cost per user each month regardless of user behavior
  - Includes entitlements to some other Microsoft products and services
  - May include Windows Server host licensing (depends on specific license)

- Per-user access pricing:
  - Grants Azure Virtual Desktop access rights for external users only
  - Pay-as-you-go through an Azure meter
  - Cost per user each month depends on user behavior
  - Only includes access rights to Azure Virtual Desktop

>   - Only supports Windows Client host licensing

Reviewer: I think these two additions may confuse more than they clarify. Correct me if I'm wrong, but I don't think any of the M365 licenses include entitlement to Windows Server session hosts with AVD. If I'm reading the [licensing prerequisites table](https://docs.microsoft.com/en-us/azure/virtual-desktop/prerequisites#operating-systems-and-licenses) correctly, Windows Server session hosts always require an RDS CAL or RDS User Subscription License. I suggest removing these two additions; the callout below seems pretty explicit to me.

In future perhaps we should ensure that per-user access pricing is represented in our licensing prerequisites table, too.

Author: ok, that's valid.  It will be valid for the other PR as well then. ;)

Let me make the change.

> [!IMPORTANT]

> > Per-user access pricing only supports the use of Windows 10 Multi-session and Windows 11 Multi-session hosts.  Server OS hosts are not supported for the per-user access option.

Reviewer: Minor tweaks:
```suggestion
> Per-user access pricing only supports the use of Windows 10 Enterprise multi-session and Windows 11 Enterprise multi-session. Windows Server session hosts are not currently supported with per-user access pricing.
```
